Elections.h

```diff
@@ -0,0 +1,123 @@
+//
+// Created by smark on 11.11.2022.
+//
+
+#ifndef LAB5_ELECTIONS_H
+#define LAB5_ELECTIONS_H
+#include <string>
+#include <iostream>
+
+namespace E {
+    using namespace C;
+    using namespace std;
+    class Elections {
+    private:
+        Candidate cand1;
+        Candidate cand2;
+        Candidate cand3;
+        Candidate cand4;
+        Candidate cand5;
+    public:
+        void printAll();
+        void setCandidates();
+        void countRates();
+        void whoWins();
```

Comment:
Please change to

        void print_all();
        void set_candidates();
        void count_rates();
        void who_wins();